changelogs/fragments/9225-proxmox-module-refactoring.yml

```
@@ -0,0 +1,9 @@
+minor_changes:
+  - proxmox - refactors the proxmox module (https://github.com/ansible-collections/community.general/pull/9225)
+  - proxmox - fixes idempotency of template conversions (https://github.com/ansible-collections/community.general/pull/9225,https://github.com/ansible-collections/community.general/issues/8811).
+  - proxmox - fixes issues with disk_volume variable (https://github.com/ansible-collections/community.general/pull/9225,https://github.com/ansible-collections/community.general/issues/9065).
+  - proxmox - fixes incorrect parsing for bind-only mounts (https://github.com/ansible-collections/community.general/pull/9225,https://github.com/ansible-collections/community.general/issues/8982).
+  - proxmox module utils - fixes ignoring of ``choose_first_if_multiple`` argument in ``get_vmid``. (https://github.com/ansible-collections/community.general/pull/9225)
```

Hi @Lithimlin sorry for being so picky about this, but thee lines are later published as the changelogs for the version - you can see what they will turn into at:
https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html

So we have these "editorial" standards that we follow. In these lines:
```suggestion
  - proxmox - fixes idempotency of template conversions (https://github.com/ansible-collections/community.general/pull/9225, https://github.com/ansible-collections/community.general/issues/8811).
  - proxmox - fixes issues with disk_volume variable (https://github.com/ansible-collections/community.general/pull/9225, https://github.com/ansible-collections/community.general/issues/9065).
  - proxmox - fixes incorrect parsing for bind-only mounts (https://github.com/ansible-collections/community.general/pull/9225, https://github.com/ansible-collections/community.general/issues/8982).
  - proxmox module utils - fixes ignoring of ``choose_first_if_multiple`` argument in ``get_vmid`` (https://github.com/ansible-collections/community.general/pull/9225).
```